.gitignore

```diff
@@ -18,6 +18,9 @@
 
 # Ignore application configuration
 /config/application.yml
+/public
+Rakefile
+Gemfile.lock
```

Comment:
You shouldn't exclude Gemfile.lock or Rakefile, these lock a certain version of dependencies and add tasks, respectively.